### PR TITLE
{bio}[foss/2019a,fosscuda/2019a] EMAN2 v2.3 w/ Python 2.7.15

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/e/EMAN2/EMAN2-2.3-foss-2019a-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+configopts = '-DENABLE_EMAN_CUDA=OFF -DENABLE_SPARX_CUDA=OFF '
+configopts += '-DPYTHON_INCLUDE_PATH="$EBROOTPYTHON/include/python%(pyshortver)s" '
+
+sanity_check_paths = {
+    'files': ['bin/e2proc2d.py', 'bin/e2proc3d.py', 'bin/e2bdb.py', 'bin/e2iminfo.py', 'bin/e2display.py',
+              'bin/e2filtertool.py'],
+    'dirs': ['doc', 'examples', 'fonts', 'images', 'lib', 'recipes', 'test/rt', 'utils']
+}
+
+modextrapaths = {'PYTHONPATH': 'lib/python%(pyshortver)s/site-packages'}
```
Comment (boegel) [inline easybuild/easyconfigs/e/EMAN2/EMAN2-2.3-foss-2019a-Python-2.7.15.eb]: The correct path is without `%(pyshortver)s`, see fix in https://github.com/akesandgren/easybuild-easyconfigs/pull/14

There's something fishy with `import EMAN2` though...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/673aeabfa66ab2e87668034dc38b9a71 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node3156.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/b20e8d313528eb3d8210f480c3d6a748 for a full test report.

File: easybuild/easyconfigs/e/EMAN2/EMAN2-2.3_fix_bad_use_cx11_abi_setting.patch [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Don't use -D_GLIBCXX_USE_CXX11_ABI=0, the code will be broken.
```
Comment (boegel) [inline easybuild/easyconfigs/e/EMAN2/EMAN2-2.3_fix_bad_use_cx11_abi_setting.patch]: @akesandgren Maybe you should elaborate a bit, and mention how things are broken, i.e. which errors are hit? That'll make it a lot easier to find this again if we'd run into the same problem again later...

Also, was this reported upstream? If so, please include a reference to the issue

```
Don't use -D_GLIBCXX_USE_CXX11_ABI=0, the code will be broken:

$ python -c 'import EMAN2'
...
TypeError: No registered converter was able to produce a C++ rvalue of type std::string from this Python object of type str
```

Comment (akesandgren) [reply easybuild/easyconfigs/e/EMAN2/EMAN2-2.3_fix_bad_use_cx11_abi_setting.patch]: No, not reported upstream. They insist on only building with conda, and i suspect they are using the same flag for their boost-python setup.

Comment (boegel) [reply easybuild/easyconfigs/e/EMAN2/EMAN2-2.3_fix_bad_use_cx11_abi_setting.patch]: 🤦‍♂ 
OK, fine

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/5da142b83a4796d2bd1f17f7c5ea288a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3115.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/34ed266b56768b5a89e0c177cac65d15 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/c270350442887c45652e6132b8a5b76a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**